porcupine/plugins/git_status.py

```diff
@@ -147,13 +147,17 @@ def _handle_queue(self) -> None:
             if not self.tree.contains_dummy(dir_id):
                 tags_changed = False
                 for item_id in self.tree.get_children(dir_id):
-                    if self._set_tag(item_id, self._choose_tag(get_path(item_id))):
-                        tags_changed = True
+                    item_path = get_path(item_id)
+                    if item_path is not None:
+                        if self._set_tag(item_id, self._choose_tag(item_path)):
+                            tags_changed = True
```

Comment:
This is already under `if not self.tree.contains_dummy(dir_id)`, so we don't get dummy items here.

```suggestion
                    assert item_path is not None
                    if self._set_tag(item_id, self._choose_tag(item_path)):
                        tags_changed = True
```